src/enums/checkers.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+const checkers = require('require-all')({
+  dirname: path.join(__dirname, '../', 'checkers'),
+  excludeDirs: 'decorators',
+  filter: /^((?!index).*)\.js$/,
+});
+
+module.exports = Object.freeze(checkers);
```

Comment:
filtering looks exactly what is required but lets do not introduce and more `enum` entry since it is:

1. a new one entry type
2. usually `enum` is predefined set of constants but in this case it is generated in runtime

so lets move this logic inside `/src/checkers/index.js` and we are good to do.

btw lets cover such filtering with unit tests.